kernel/cClosure.ml

```diff
@@ -868,14 +746,15 @@ let drop_parameters depth n argstk =
   anomaly (Pp.str "ill-typed term: found a match on a partially applied constructor.")
 
 let inductive_subst mib u pms =
-  let rec mk_pms i ctx = match ctx with
-  | [] -> subs_id 0
-  | RelDecl.LocalAssum _ :: ctx ->
-    let subs = mk_pms (i - 1) ctx in
-    subs_cons pms.(i) subs
-  | RelDecl.LocalDef (_, c, _) :: ctx ->
-    let subs = mk_pms i ctx in
-    subs_cons (mk_clos (subs,u) c) subs
+  let rec mk_pms i ctx =
+    match ctx with
+    | [] -> subs_id 0
+    | RelDecl.LocalAssum _ :: ctx ->
+        let subs = mk_pms (i - 1) ctx in
+        subs_cons pms.(i) subs
+    | RelDecl.LocalDef (_, c, _) :: ctx ->
+        let subs = mk_pms i ctx in
+        subs_cons (mk_clos (subs,u) c) subs
```

Comment:
What happened here?

Reply:
I would say that I fixed indentation.

Reply:
The match body is incorrectly indented as I mentioned in another comment.
I don't know if any auto indenters produce the style where the patterns get 0 additional identation but when it appears it is usually deliberate.